src/istgt_lu_ctl.c

```diff
@@ -26,6 +26,7 @@
  */
 
 #ifdef HAVE_CONFIG_H
+#include <json-c/json_object.h>
```

Comment:
Thre is no need of `HAVE_CONFIG_H` macro protection fo `json_object.h` header file.